common/evaluators/reuters_evaluator.py

```diff
@@ -0,0 +1,28 @@
+import torch
+import torch.nn.functional as F
+import numpy as np
+
+from .evaluator import Evaluator
+
+
+class ReutersEvaluator(Evaluator):
+
+    def get_scores(self):
+        self.model.eval()
+        self.data_loader.init_epoch()
+        n_dev_correct = 0
+        total_loss = 0
+
+        for batch_idx, batch in enumerate(self.data_loader):
+            scores = self.model(batch.text)
+            # Using binary accuracy
+            for tensor1, tensor2 in zip(scores.round().long().data, batch.label.data):
+                print(tensor1, tensor2)
```

Comment:
remove this

Reply:
My bad. I made the change, forgot to push it.